firebase-functions/ktx/src/test/kotlin/com/google/firebase/functions/ktx/FunctionsTests.kt

```diff
@@ -99,7 +99,7 @@ class LibraryVersionTest : BaseTestCase() {
   @Test
   fun `library version should be registered with runtime`() {
     val publisher = Firebase.app.get(UserAgentPublisher::class.java)
-    assertThat(publisher.userAgent).contains(LIBRARY_NAME)
+    assertThat(publisher.userAgent).contains("fire-fun-ktx")
```

Comment:
This only happens if the ktx sdk is present, right? or should it work even if there's no common-ktx artifact?

Reply:
currently i think with the new approach irrespective of anything this will be logged.. I think we should stop supporting this.